app/src/main/kotlin/com/wire/android/ui/home/HomeScreen.kt

```diff
@@ -83,6 +83,7 @@ fun HomeScreen(
     homeViewModel.checkRequirements()
     LaunchedEffect(Unit) {
         featureFlagNotificationViewModel.loadInitialSync()
+        featureFlagNotificationViewModel.initialSync()
```

Comment:
Do we need both calls?

From `FeatureFlagNotificationViewModel`, `loadInitialSync` is calling `initialSync` from the vm scope 🤔

Reply:
good catch! Forgot to remove one! Thanks